### PR TITLE
Update api-extensions.md

### PR DESCRIPTION
**Pull request by goenkale**
Added: how to add the custom permission mapping using Bonita Studio

### CODE REVIEW

**Comment by tbouffard** [comment]
@goenkale thanks for your contribution
Can you please sign the CLA to allow us to review and integrate your work?

File: md/api-extensions.md [hunk 1]
```diff
@@ -57,12 +57,18 @@ A REST API extension example resource and API extension viewer page are provided
...
 * Edit `platform_conf/current/tenants/<TENANT_ID>/tenant_portal/custom-permissions-mapping.properties`.
 * Add the following line: `profile|User=[demoPermission]`.
 * Push the changed platform configuration into database using the _setup tool_.
 * Restart Tomcat bundle
 
+In Bonita Studio:
+* Open the `custom-permissions-mapping.properties` from Development menu > REST API extension > Edit permission mapping
+* Add the following line: `profile|User=[demoPermission]`
+* Save the file
+* Logout from the current session and relogin
```
Comment (educhastenier) [inline md/api-extensions.md]: ~relogin~ :arrow_right: log back in

File: md/api-extensions.md [hunk 2]
```diff
@@ -57,12 +57,18 @@ A REST API extension example resource and API extension viewer page are provided
 
 To use the examples, define a [custom permissions](rest-api-authorization.md#custom-permissions-mapping) mapping for `demoPermission`:
 
+In a bundle:
```
Comment (educhastenier) [inline md/api-extensions.md]: In a Tomcat / WildFly bundle:

**Comment by educhastenier** [comment]
test this please

**Comment by tnguyen1** [comment]
retest this please

**Pull request merged**